CSS-Folder/Download.css

```diff
@@ -0,0 +1,132 @@
+.Download-page {
+  font-family: "Actor", sans-serif;
+  background-color: #17082a;
+  margin: 0;
+  /* width: 375px; */
+  /* height: 812px; */
+}
+.header img {
+  width: 100%;
+}
+
+
+.Movie-name {
+  display: flex;
+  font-size: 6vw;
+  align-items: center;
+  letter-spacing: 0.1vw;
+  margin: 11vw 2.5vw;
+  font-style: normal;
+  font-weight: normal;
+}
+
+.main-content {
+  color: white;
+}
+.Quality-section {
+  background-color: #17082a;
+  margin: 4vw;
+  height: 50vw;
+  letter-spacing: 0.38px;
+}
+.Movie-Quality {
+  opacity: 30%;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 5vw;
+  letter-spacing: 0.2vw;
+
+}
+
+.Quality-box-pick{
+display: flex;
+justify-content: space-between;
+/* width:88%; */
+margin: 0vw 2vw 0vw 9vw;
+}
+
+.Quality-box {
+  text-decoration: none;
+  text-align: center;
+  font-size: 4vw;
+  color: white;
+  height: 39vw;
+  width:23vw;
+  background: #200f37;
+  border-radius: 5vw;
+  line-height: 36vw;
+ 
+
+}
+
+.Quality-box:hover {
+border: solid 0.5vw #FF8FC6 ;
+
+
+}
+
+.Voice-acting {
+  /* display: flex; */
+  /* flex-direction: column; */
+  margin: 0vw 8vw;
+}
+
+
+.languges-style{
+  display:flex;
+  justify-content: space-between;
+  align-content: center;
+  margin: 8vw 0vw;
+  font-size: 5vw;
+  font-style: normal;
+font-weight: normal;
+letter-spacing: -0.1vw;
+}
+
+
+/* ------------------How to control Checbox Style--------------- */

```

Comment:
We must figure out away to do this

Reply:
this the solution bro
`input[type=checkbox] {
  -webkit-appearance: none;
  -moz-appearance: none;

  width: 10vw; /*the box width */
  height: 10vw; /*the box height */
  border: .01vw solid rgb(255, 255, 255);  /*the box-border weight */

  border-radius: 3vw;
  opacity: 0.1;

}

/* For the box after check (not the arrow) */
input[type=checkbox]:checked {
  background-color: #5442B6;
  opacity: 1;
  border: none;
  color:rgb(255, 255, 255);
  display: flex;
  justify-content: center;
  align-content: center;
  padding-top: 1.5vw;

}

input[type=checkbox]:checked:after {
  display: inline-block;

  width: 2vw; /*the ✔ sign length */
  height: 4vw;    /*the ✔ sign length */
  border: solid white;
  border-width: 0 0.8vw 0.8vw 0; /*the ✔ sign weight */

  -webkit-transform: rotate(45deg); /*rotation to make ✔ sign  */
  -moz-transform: rotate(45deg);/*rotation to make ✔ sign  */
  -ms-transform: rotate(45deg);/*rotation to make ✔ sign  */
  transform: rotate(45deg); /*rotation to make ✔ sign  */

  content: ""; /*what to type after click */


}
`